dependencies/Mapbox.IO.Compression-unity/Mapbox.IO.Compression.sln

```diff
@@ -0,0 +1,65 @@
+﻿
```

Comment:
Please delete this file.
It's a leftover from when we had `Mapbox.IO.Compression-unity` as a submodule.